docs/testing/troubleshoot-manifest.md

```diff
@@ -1,73 +1,54 @@
 
-## Validate your manifest with the Office Add-in Validator
+```command&nbsp;line
+npm run validate
+```
 
-To help ensure that the manifest file that describes your Office Add-in is correct and complete, validate it against the [Office Add-in Validator](https://github.com/OfficeDev/office-addin-validator).
+![Animated gif that shows the Yo Office validator being run at the command line and generating results that show Validation Passed](../images/yo-office-validator.gif)
 
-### To use the Office Add-in Validator to validate your manifest
+> [!NOTE]
+> To have access to this functionality, your add-in project must have been created by using [Yeoman generator for Office Add-ins](https://www.npmjs.com/package/generator-office) version 1.1.17 or later.
 
-1. Install [Node.js](https://nodejs.org/download/). 
+## Validate your manifest in a custom project
 
-2. Open a command prompt / terminal as an administrator, and install the Office Add-in Validator and its dependencies globally by using the following command:
+If your project was created without using yo office, you can still validate the manifest using the office-toolbox package. This is the same package used by yo office for validation.
 
-	```command&nbsp;line
-	npm install -g office-addin-validator
-	```
-	
-	> [!NOTE]
-	> If you already have Yo Office installed, upgrade to the latest version, and the validator will be installed as a dependency.
+1. Install [Node.js](https://nodejs.org/download/).
 
-3. Run the following command to validate your manifest. Replace MANIFEST.XML with the path to the manifest XML file.
+2. Open a command prompt / terminal as an administrator, and run the Office Add-in validator on your manifest file. The Office Add-in validator is in the office-toolbox package.
 
 	```command&nbsp;line
-	validate-office-addin MANIFEST.XML
+	npx office-toolbox validate -m manifest.xml
 	```
 
 ## Validate your manifest against the XML schema
 
-To help ensure that the manifest file follows the correct schema, including any namespaces for the elements you are using. If you copied elements from other sample manifests double check you also **include the appropriate namespaces**. You can validate a manifest against the [XML Schema Definition (XSD)](https://github.com/OfficeDev/office-js-docs-pr/tree/master/docs/overview/schemas) files. You can use an XML schema validation tool to perform this validation. 
-
-
+To help ensure that the manifest file follows the correct schema, including any namespaces for the elements you are using. If you copied elements from other sample manifests double check you also **include the appropriate namespaces**. You can validate a manifest against the [XML Schema Definition (XSD)](https://github.com/OfficeDev/office-js-docs-pr/tree/master/docs/overview/schemas) files. You can use an XML schema validation tool to perform this validation.
```

Comment:
I can't spot what you changed in this paragraph, but the opening sentence here seems malformed/incomplete. Do you want to fix that as part of this PR?

Reply:
@kbrandl I put in a rewrite for this. See if this works!